packages/create-strapi-app/README.md

```diff
@@ -0,0 +1 @@
+# Create strapi app
```

Comment:
We can maybe add a short description for this package?

Reply:
Like, the way to use this package for example.

Reply:
Done